velox/exec/tests/utils/OperatorTestBase.h

```diff
@@ -35,6 +35,10 @@ class OperatorTestBase : public testing::Test,
 
   void SetUp() override;
 
+  // This function allows tests derived from this base to register
```

Comment:
Comments must end with a period.

Also, consider shortening for clarity: "Allow base classes to register custom vector serde. By default, registers Presto-compatible serde."